test/service/test_response.py

```diff
@@ -1743,6 +1743,133 @@ def test_pagination_search_after_search_before(self):
         self.assertEqual(second_page_previous['search_before_uid'], 'doc#308eea51-d14b-4036-8cd1-cfd81d7532c3')
 
 
+class TestResponseInnerEntitySamples(WebServiceTestCase):
+    maxDiff = None
+
+    @classmethod
+    def bundles(cls) -> List[BundleFQID]:
+        return super().bundles() + [
```

Comment:
It definitely doesn't hurt to inherit the bundles used in the base class, but if you don't need them for this test, it may run faster with a smaller subset. On the other hand, more bundles == more comprehensive test :shrug:

Reply:
The base class brings in bundle `aaa96233` which allows the subtest for `{'sampleEntityType': {'is': ['specimens']}}` to return 2 hits instead of 1 like the other subtests. I feel this is a useful aspect of the test case to have, especially since the hits are for projects that contain a different number of samples from each other.